reframe/frontend/cli.py

```diff
@@ -769,8 +769,11 @@ def restrict_logging():
         if config_param == 'all':
             printer.info(str(rt.site_config))
         else:
-            value = rt.get_option(config_param)
-            if value is None:
+            # Create a unique value to differentiate between configuration
+            # parameters with value `None` and invalid ones
+            default = time.time()
+            value = rt.get_option(config_param, default)
+            if value == default:
```

Comment:
Nice trick, but I would rather use another type of unique value:

```suggestion
            default = {'token'}
            value = rt.get_option(config_param, default)
            if value is default:
```